lib/smblib/find_password.c

```diff
@@ -55,7 +55,7 @@ int next_password(char *pw, int pwlen)
     if (pwinit == FALSE) {
 
         pwinit = TRUE;
-        memset(pw, 0, pwlen + 1);
+        memset_s(pw, pwlen + 1, 0, pwlen + 1);
```

Comment:
I don't think we should be doing any changes in this library. It is deprecated and as you can see from the list of protocol names at the top of this file only used for protocols obsolete since mid-1990's which are transferring their password in clear-text around the network.

Reply:
true